examples/pubsub/project/main.tf

```diff
@@ -31,12 +31,12 @@ module "log_export" {
 }
 
 module "destination" {
-  source                   = "../../..//modules/pubsub"
-  project_id               = var.project_id
-  topic_labels             = { topic_key : "topic_label" }
-  topic_name               = "pubsub-project-${random_string.suffix.result}"
-  log_sink_writer_identity = module.log_export.writer_identity
-  create_subscriber        = true
-  subscription_labels      = { subscription_key : "subscription_label" }
+  source                                  = "../../..//modules/pubsub"
```

Comment:
```suggestion
  source                                  = "../../../modules/pubsub"
```

Reply:
> 

Still failing. I've tested this with double slashes and it worked on my personal project. Very weird, but it's failing with no good reason.